### ISSUE TITLE
TuYA TS0601_smart_air_house_keeper "Unrecognized DP"

### ISSUE THREAD

**Issue by FHannes** [Koenkk/zigbee-herdsman-converters#4189]
I recently added the following device to my network: https://www.aliexpress.com/item/1005003787375272.html

It was recognized as the [TuYA TS0601_smart_air_house_keeper](https://www.zigbee2mqtt.io/devices/TS0601_smart_air_house_keeper.html#tuya-ts0601_smart_air_house_keeper) and seems to be mostly working with only the formaldehyd measurement showing "N/A ppm". But Zigbee2MQTT keeps reporting an error with the device:
```
debug Received Zigbee message from '0x0123456789abcdef', type 'commandDataReport', cluster 'manuSpecificTuya', data '{"dpValues":[{"data":{"data":[0,0,0,4],"type":"Buffer"},"datatype":2,"dp":20}],"seq":512}' from endpoint 1 with groupID 0
warning zigbee-herdsman-converters:TuyaSmartAirBox: Unrecognized DP #20 with data {"dp":20,"datatype":2,"data":{"type":"Buffer","data":[0,0,0,4]}}
debug Received Zigbee message from '0x0123456789abcdef', type 'commandDataReport', cluster 'manuSpecificTuya', data '{"dpValues":[{"data":{"data":[0,0,0,5],"type":"Buffer"},"datatype":2,"dp":21}],"seq":512}' from endpoint 1 with groupID 0
```

**Comment by Koenkk** [comment]
May already be fixed in z2m 1.25.1 (just released)

**Comment by FHannes** [comment]
I can confirm it's been resolved. Thank you 👍
